gce-wake.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+# Start all the instances in the specified GCE deployment
+  - hosts: localhost
+    tasks:
+      - name: Deploy gce.py
+        file: src=gce.py dest=./gce.py mode=0744
+      - name: Deploy gce_account.pem
+        file: src=gce_account.pem dest=./gce_account.pem mode=0600
```

Comment:
Gotcha. That sounds reasonable. If anyone is using an unqualified path in their config they should speak up now.